src/components/Publication/PublicationBody.tsx

```diff
@@ -34,7 +34,7 @@ const PublicationBody: FC<Props> = ({ publication }) => {
         </div>
       )}
       {publication?.metadata?.media?.length > 0 ? (
-        <Attachments attachments={publication?.metadata?.media} />
+        <Attachments attachments={publication?.metadata?.media} publication={publication} />
```

Comment:
**suggestion:** we can move attachments props inside the `Attachments` component